step/errors.go

```diff
@@ -62,6 +109,9 @@ func findXcodebuildErrors(out string) []string {
 		return nil
 	}
 
+	// Prefer NSErrors if found for all errors,
+	// this is because an NSError has a suggestion in addition to the error reason,
+	// but we use regular expression for parsing NSErrors.
 	if len(nserrors) == len(errorLines) {
```

Comment:
If I understand this code correctly: if one NSError parsing fails -> list length will be different -> entire block is skipped -> even the successfully parsed NSErrors are ignored

Reply:
True, because NSErrors seems to have regular error pairs and the regular error parsing is more reliable.